__tests__/app.test.js

```diff
@@ -222,3 +223,78 @@ describe ('POST /api/articles/:article_id/comments', () => {
       })
     })
   });
+
+
+
+describe('8. PATCH /api/articles/:article_id', () =>{
+ test('201 - positive vote change adds votes to count', () => {
+    return request(app)
+    .patch('/api/articles/1')
+    .send({inc_votes : 100})
+    .expect(201)
```

Comment:
Expect a 200 - 201 indicates something has been created whereas we just want confirmation that our patch request has done what we asked.